.github/workflows/macos_dependencies.txt

```diff
@@ -1,7 +1,7 @@
 blas
 cairo
-clangxx_osx-arm64
```

Comment:
You may revert this unrelated change. (It was sorted alphabetically).

Reply:
I also used the F9 sort in VSCode + delete duplicate lines when doing the dozen or so rebases.. to simplify it I just kept that order in the meantime.
Was yours sorted by hand or automatically?

Reply:
Automatically. I suppose there is a different sorting algorithm for non-alphabetical/digit characters.